src/lib/Hydra/Schema/Projects.pm

```diff
@@ -252,7 +252,10 @@ my %hint = (
         "displayname",
         "description",
         "homepage",
-        "owner"
+        "owner",
+        "declfile",
+        "decltype",
+        "declvalue"
```

Comment:
It would be a bigger lift, but it might be nicer if the declarative configuration here were a JobsetInput type, and thus a nested value here. What do you think?

Reply:
It would have to be its own type because JobsetInput has an `emailresponsible` field. I created its own type in 43cbe6693e7dc39612999f193b103d87137710dc, but reverted it in later fixups (to see how both would look) -- thoughts?